CHANGELOG.md

```diff
@@ -9,9 +9,10 @@
 * Farming Simulator 19 (2018) - Added support (By @Vito0912 #531)
 * Assetto Corsa - Fixed how `state.numplayers` is set (By @podrivo #538)
 * TeamSpeak 2 - Fixed how `state.name` is set (By @podrivo #544)
-* Grand Theft Auto: San Andreas OpenMP - Fixed `state.players` returning an empty array (By @Focus04 #547) 
+* Grand Theft Auto: San Andreas OpenMP - Fixed `state.players` returning an empty array (By @Focus04 #547)
```

Comment:
seems like some stray whitespaces were modified, can you remove em?

Reply:
Just to confirm you want me to re-add stray whitespaces?

Reply:
Yes, I meant to revert them, sorry (:

Reply:
No problem I will try to revert. I have never had a request to keep stray white spaces before and it's a standard thing for VS Code to remove them.

Reply:
Trailing white-spaces have now been re-added